dotcom-rendering/src/lib/renderElement.tsx

```diff
@@ -490,15 +491,69 @@ export const renderElement = ({
 				</Island>
 			);
 		case 'model.dotcomrendering.pageElements.MediaAtomBlockElement':
-			return (
-				<VideoAtom
-					format={format}
-					assets={element.assets}
-					poster={element.posterImage?.[0]?.url}
-					caption={element.title}
-					isMainMedia={isMainMedia}
-				/>
-			);
+			/*
+				- MediaAtomBlockElement is used for self-hosted videos
+				- Historically, these videos have been self-hosted for legal or sensitive reasons
+					- These videos play in the `VideoAtom` component
+				- Looping videos, introduced in July 2025, are also self-hosted
+					- Thus they are delivered as a MediaAtomBlockElement
+					- However they need to display in a different video player
+					- This is handled in the new `LoopVideoInArticle` component
+				- We need to differentiate between the two forms of video
+					- We can do this by interrogating the atom's metadata, which includes the new attribute `videoPlayerFormat`
+
+				- Note: we'll probably extend this functionality to handle new 'Cinemagraph' videos
+					- These may use the looping video, or yet another new, video player
+					- But they will still be Media Atoms
+			*/
+
+			if (element?.videoPlayerFormat === 'Loop') {
+				const posterImageExists = !!element.posterImage?.[0]?.url;
+				return (
+					<>
+						{posterImageExists && (
+							<Island
+								priority="critical"
+								defer={{ until: 'visible' }}
+							>
+								<LoopVideoInArticle
+									assets={element.assets}
+									atomId={element.id}
+									uniqueId={`${Math.random()}`}
+									height={400}
+									width={500}
+									posterImage={
+										element.posterImage?.[0]?.url ?? ''
+									}
+									fallbackImage={
+										element.posterImage?.[0]?.url ?? ''
+									}
+									fallbackImageSize="small"
+									fallbackImageLoading="lazy"
+									fallbackImageAlt={element.title}
+									fallbackImageAspectRatio="5:4"
+									linkTo="Article-embed-MediaAtomBlockElement"
+									format={format}
+									caption={element.title}
+									isMainMedia={isMainMedia}
+									subtitleSize={element.subtitleSize}
+									subtitleSource={element.subtitleSource}
```

Comment:
We should be able to grab the relevant asset with something like this:

```
const getSubtitleAsset = (assets: VideoAssets[]) => {
	// Get the first subtitle asset from assets with a mimetype of 'text/vtt'
	return assets.find((asset) => asset.mimeType === 'text/vtt');
}
```

Then:

```suggestion
									subtitleSource={getSubtitleAsset(assets)?.url}
```

Reply:
Done!